### DISCUSSION TITLE
Show suggestions tool tip when hover on annotations.

### DISCUSSION THREAD

**Discussion by saikiranravupalli** [ajaxorg/ace#5082] [Q&A]
We kept js lint for the editor, so we are getting annotations(underlines) for the code line. 
When we hover on the warning icon/error icon then a tool tip will be displayed. We are trying to show a tooltip when hovering on the marker layer(underline).
![image](https://user-images.githubusercontent.com/59285481/222898140-b9f449d5-e683-4a3c-988b-3b0936232daf.png)
![image](https://user-images.githubusercontent.com/59285481/222898089-5a1c2a32-e204-44e3-8518-3dfa7654ab08.png)

Is that possible? How can we achieve that?

**Comment by akoreman** [comment]
Hi, thanks for your question. Currently, there is no option to add tooltips when hovering over markers. We have an [issue](https://github.com/ajaxorg/ace/issues/2720) requesting this feature if you want to leave your upvote.
